### PR TITLE
docs(providers): Instructions for how to use the Google Drive backend.

### PR DESCRIPTION
**Pull request by xkxx**
Hi @jkowalski,

This is some documentation for how to use the new Google Drive backend.

I actually have screenshots for the step-by-step instructions, it would significantly lower the barrier for non-technical users to set this up. I'm not sure how you'd like to organize these image assets though. Please let me know.

Thanks,

xkxx

### CODE REVIEW

File: site/content/docs/Repositories/_index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ A repository is a place where Kopia stores its snapshot data. Kopia currently su
 * [Azure Blob Storage](#azure)
 * [Amazon S3](#amazon-s3) (and compatible)
 * [Backblaze B2](#b2)
+* [Google Drive](#google-drive)
```
Comment (jkowalski) [inline site/content/docs/Repositories/_index.md:13]: can we clearly mark this as experimental?

**Comment by jkowalski** [comment]
You should be able to put images in the same directory as the MD file and just include them. There are some png and svg files under Architecture already.

File: site/content/docs/Repositories/_index.md (hunk 1 above)
Comment (xkxx) [reply site/content/docs/Repositories/_index.md:13]: Done. I added a note under the Google Drive section.

**Comment by xkxx** [comment]
Got it. I added screenshots.

**Pull request merged**